### DISCUSSION TITLE
I cant Push!

### DISCUSSION THREAD

**Discussion by dami54ano** [community/community#132248] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If I push my GitHub (desktop), I need to sign in, but after that, nothing happens, and I don't get redirected. "

You are being redirected to the authorized application.
If your browser does not redirect you back, please visit [this setup page](x-github-desktop-auth://oauth?code=9b193afb400517371700&state=b1d9e24b-53e6-4015-b4f3-6e220e184406) to continue." but this link doesnt work. So idk what to do.

Thank you

**Comment by SajanGhimire1** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> If I push my GitHub (desktop), I need to sign in, but after that, nothing happens, and I don't get redirected. "
> 
> You are being redirected to the authorized application. If your browser does not redirect you back, please visit this setup page to continue." but this link doesnt work. So idk what to do.
> 
> Thank you

Hi evening,
1. Update GitHub Desktop: Ensure it's the latest version.
2. Sign Out and In: Go to File > Options > Sign out, then sign back in.
3. Clear Browser Cache: Clear your browser's cache and cookies.
4. Use a Different Browser: Open the authorization link in another browser.
5. Manual Authentication: Use Sign in with browser in GitHub Desktop, copy the URL if redirection fails.
6. Personal Access Token (PAT):
-Generate a PAT from [GitHub Tokens](https://github.com/settings/tokens).
-Use the PAT to sign in on GitHub Desktop.

**Reply by dami54ano** [reply]
Thank you, I will try it later. I have also tried some of them.

**Comment by rrsayao** [comment]
Might also be worth it to check check if git didn't open a pop up window asking for your authorization. Either that or a new tab on your browser.
